sdk/src/account/private_key.rs

```diff
@@ -30,43 +30,45 @@ use std::{
 };
 
 #[pyclass(frozen)]
-#[derive(Clone, PartialEq, Eq)]
+#[derive(Copy, Clone, PartialEq, Eq)]
 pub struct PrivateKey(PrivateKeyNative);
 
-impl Deref for PrivateKey {
-    type Target = PrivateKeyNative;
-
-    fn deref(&self) -> &Self::Target {
-        &self.0
-    }
-}
-
 #[pymethods]
 impl PrivateKey {
     /// Generates a new private key using a cryptographically secure random number generator
+    #[allow(clippy::new_without_default)]
     #[new]
-    pub fn random() -> Self {
```

Comment:
Not matter how it’s named, from python it can be called only by `PrivateKey()`. So why need to rename this method and suppress clippy?

Reply:
Mainly because `new()` is the idiom we want to keep across the Aleo SDKs for new objects. However, we can implement Default() instead of suppressing the lint for good hygiene.